API.md

```diff
@@ -11,118 +11,120 @@
         -   [Parameters][7]
     -   [query][8]
         -   [Parameters][9]
-    -   [setInput][10]
-        -   [Parameters][11]
-    -   [setProximity][12]
-        -   [Parameters][13]
-    -   [getProximity][14]
-    -   [setRenderFunction][15]
-        -   [Parameters][16]
-    -   [getRenderFunction][17]
-    -   [setLanguage][18]
-        -   [Parameters][19]
-    -   [getLanguage][20]
-    -   [getZoom][21]
-    -   [setZoom][22]
-        -   [Parameters][23]
-    -   [getFlyTo][24]
-    -   [setFlyTo][25]
-        -   [Parameters][26]
-    -   [getPlaceholder][27]
-    -   [setPlaceholder][28]
-        -   [Parameters][29]
-    -   [getBbox][30]
-    -   [setBbox][31]
-        -   [Parameters][32]
-    -   [getCountries][33]
-    -   [setCountries][34]
-        -   [Parameters][35]
-    -   [getTypes][36]
-    -   [setTypes][37]
-        -   [Parameters][38]
-    -   [getMinLength][39]
-    -   [setMinLength][40]
-        -   [Parameters][41]
-    -   [getLimit][42]
-    -   [setLimit][43]
-        -   [Parameters][44]
-    -   [getFilter][45]
-    -   [setFilter][46]
-        -   [Parameters][47]
-    -   [setGeocoderApi][48]
-        -   [Parameters][49]
-    -   [getGeocoderApi][50]
-    -   [on][51]
-        -   [Parameters][52]
-    -   [off][53]
-        -   [Parameters][54]
+    -   [\_fitBoundsForMarkers][10]
```

Comment:
are these trailing characters `\_` intentional since it kind of signal that it's private but part of API.md?

Reply:
Whoops forgot to add a private tag so this wouldnt show up